.bumpversion.cfg

```diff
@@ -1,8 +1,8 @@
 [bumpversion]
-current_version = 0.4.0
+current_version = 0.4.4
```

Comment:
This is because the previous releases hadn't used bumpversion so we're skipping ahead a bit.

Reply:
Fast forward into a brighter and better future! 🚀 May bumpversion be with us!